test/unit/browser/pages/vscode.test.ts

```diff
@@ -175,4 +185,171 @@ describe("vscode", () => {
       localStorage.removeItem("colorThemeData")
     })
   })
+  describe("getConfigurationForLoader", () => {
+    beforeEach(() => {
+      const { window } = new JSDOM()
+      global.document = window.document
+    })
+    afterEach(() => {
+      global.window = undefined as unknown as Window & typeof globalThis
+      global.document = undefined as unknown as Document & typeof globalThis
```

Comment:
We shouldn't need any more globals right? Since we pass everything in to the individual functions now. In fact avoiding globals is probably good for testing since it'll catch any mistakes where we might accidentally use a global.

Reply:
Hmm...well we need `window` and `document` to come from somewhere. I guess the point you're making is, define them locally in each test rather than on the `global` object?

Reply:
I took your point as that and refactored to not use globals in test. Good call!